common/src/main/java/com/wynntils/services/itemfilter/statproviders/TargetStatProvider.java

```diff
@@ -9,7 +9,7 @@
 import com.wynntils.services.itemfilter.type.ItemStatProvider;
 import java.util.List;
 
-public class TargetStatProperty extends ItemStatProvider<String> {
+public class TargetStatProvider extends ItemStatProvider<String> {
```

Comment:
Should probably be done in a seperate pr but this is only matching inputs based on the annotation, not the full name destination. Can't query for "Detlas", you have to do "De"

Reply:
Yeah, I may PR this before you finish the UI. We'll see.